sdk/search/tests.yml

```diff
@@ -5,12 +5,9 @@ jobs:
     parameters:
       ServiceDirectory: search
       Timeout: 120
+      MaxParallel: 2
```

Comment:
What's the reason of having maximum of 2?

Reply:
Still have limitations on the total number of service we can be using concurrently.